src/FondOfSpryker/Client/ProductListsRestApi/ProductListsRestApiFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace FondOfSpryker\Client\ProductListsRestApi;
+
+use FondOfSpryker\Client\ProductListsRestApi\Dependency\Client\ProductListsRestApiToZedRequestClientInterface;
+use FondOfSpryker\Client\ProductListsRestApi\Zed\ProductListsRestApiStub;
+use FondOfSpryker\Client\ProductListsRestApi\Zed\ProductListsRestApiStubInterface;
+use Spryker\Client\Kernel\AbstractFactory;
+
+class ProductListsRestApiFactory extends AbstractFactory
+{
+    /**
+     * @return \FondOfSpryker\Client\ProductListsRestApi\Zed\ProductListsRestApiStubInterface
+     */
+    public function createZedProductListsRestApiStub(): ProductListsRestApiStubInterface
+    {
+        return new ProductListsRestApiStub($this->getZedRequestClient());
+    }
+
+    /**
+     * @throws
```

Comment:
Can you specify the thrown Exception type

Reply:
I can't specify the exception type. The auto formatter remove it automatically